README.md

```diff
@@ -1,226 +1,139 @@
-- nginx -> [nginx:alpine](https://hub.docker.com/r/_/nginx/)
-- percona -> [percona:latest](https://hub.docker.com/r/_/percona/)
-- php -> [srcoder/development-php:7.0-fpm](https://hub.docker.com/r/srcoder/development-php/)
-- php5 -> [srcoder/development-php:5.6-fpm](https://hub.docker.com/r/srcoder/development-php/)
-- php71 -> [srcoder/development-php:7.1-fpm](https://hub.docker.com/r/srcoder/development-php/)
-- redis -> [redis:alpine](https://hub.docker.com/r/_/redis/)
-- mailhog -> [mailhog/mailhog:latest](https://hub.docker.com/r/mailhog/mailhog/)
-- mytop -> [srcoder/mytop:latest](https://hub.docker.com/r/srcoder/mytop/)
-- ctop -> [wrfly/ctop](https://hub.docker.com/r/wrfly/ctop/)
+**OSX users** should also install:
 
+ - [Docker Sync][3] and its dependencies
+ - Coreutils using Homebrew `brew install coreutils`
 
-## Installation
+ - Make sure that port 80/443 and 3306 are not being used by other services.
+`sudo netstat -tupln|egrep '80|443|3306'`
 
-- Install [docker](https://docs.docker.com/)
-- Install docker [compose](https://docs.docker.com/compose/install/) >1.6.1
-- Clone this project
-  `git clone git@github.com:JeroenBoersma/docker-compose-development.git development`
+# How to get started
+Make sure that you have the prerequisites installed and running correctly before proceeding.
 
-## Additional installation steps for OSX
-
-- Install [Docker Sync](http://docker-sync.io/) and its dependencies.
-- You need to install coreutils using Homebrew only to have realpath command. :)
-  `brew install coreutils`
-
-## Persistent data container
-
-For running the database you need to create a new persistent data volume `docker volume create --name dockerdev-mysql-volume`
-
-If you already were using this repo before (or want a local directory), you can map the existing volume with:
-`docker volume create -o 'type=none' -o 'device='${PWD}'/mysql' -o 'o=bind' dockerdev-mysql-volume`
-
-## Preparation
-
-Tested under Linux. Also tested on OSX by a limited number of developers.
-For Windows, take a look at the docker beta (heard that good performances are met)
-
-Stop all other local Webservers running on port 80/443, stop all MySQL database servers on port 3306.
-
-Set-up your database credentials (`conf/mysql`) and Blackfire (`conf/blackfire`) profile in the conf directory.
-
-- conf/mysql (`MYSQL_ROOT_PASSWORD=something`)
-
-## Blackfire
+## 1). Configure your environment
+ 1. Clone this repository:
+```
+git clone git@github.com:JeroenBoersma/docker-compose-development.git development
+```
+ 2. Create a new persistent data volume with:
+```
+docker volume create --name dockerdev-mysql-volume
+```
+Or, if you already were using this repository before (or want a local directory), you can map the existing volume with:
+```
+docker volume create -o 'type=none' -o 'device='${PWD}'/mysql' -o 'o=bind' dockerdev-mysql-volume
+```
+ 3. Configure you MySQL credentials by editing [conf/mysql](conf/mysql). 
```

Comment:
Deleted config (accidentally did it on my own machine, still need to restore it :-()